docs/standards/nft-2.0/03-LSP8-Identifiable-Digital-Asset.md

```diff
@@ -4,37 +4,92 @@ sidebar_label: 'LSP8 - Identifiable Digital Asset'
+
+A good example where all these problems could have effects is a video game. An NFT could be used to represent a unique item like a weapon. We might want the player to have the ability to upgrade the weapon features over time, so that it becomes more powerful.
+
+## What does this Standard represent?
+
+LSP8 Identifiable Digital Asset is a standard that aims to describe _non-fungible_ assets. _Non-fungible_ means that each asset is unique and different. They are distinguishable from each other, and therefore not interchangeable.
 
-LSP8 assets are similar in nature to [ERC721](https://eips.ethereum.org/EIPS/eip-721) tokens (NFTs)
+Since every single asset is unique on its own, they are differentiated by a unique identifier: a `tokenId`. The identifier can be anything from a unique serial number to another [ERC725Y](https://github.com/ERC725Alliance/ERC725/blob/main/docs/ERC-725.md#erc725y) smart contract that contains information and metadata specific to this `tokenId`.
 
-Since every single asset is unique on its own, they are differentiated by a unique identifier: a `tokenId`. The identifier can be anything from a unique serial number, to an other [ERC725Y](https://github.com/ERC725Alliance/ERC725/blob/main/docs/ERC-725.md#erc725y) Smart contract that contain information and metadata specific to this `tokenId`.
+LSP8 solves the current problems of NFTs by using the ERC725Y standard as its base. By using a generic key-value store, an LSP8 contract comes now with the following features:
 
-## NFT Metadata
+- attach an unlimited amount of information (= metadata), making the asset more customisable and more unique.
+- know when the metadata has been altered, as a `DataChanged` event will be emitted in ERC725Y.
+- Metadata is now **hash reference** (= data reference), instead of URL reference.
 
-One of the key difference of LSP8 is that each newly created `tokenId` (NFT) has its own metadata, that describes its uniqueness. Such metadata can be a reference either:
+LSP8 assets are similar to [ERC721](https://eips.ethereum.org/EIPS/eip-721) tokens (NFTs). What makes them different is their underlying base, making them more customisable and their metadata more upgradable over time. This is what makes them a new generation of unique assets known as **NFT 2.0**.
 
-- _off-chain_ (a **JSON file url** stored on IPFS, a public / private registry, etc...)
-- _on-chain_ (an other **ERC725Y smart contract**)
+### NFT Metadata
 
-It is possible to know where the metadata for a specific tokenID is located by simply querying the LSP8 contract, giving the `tokenID` as an argument.
+One of the key differences of LSP8 is that it allows representing each NFT as more than just a unique `tokenID` number. Each newly created `tokenId` (NFT) has its own metadata, that describes its uniqueness. Such metadata can be a reference either:
```

Comment:
do you want to write `tokenId` or `tokenID` ? I saw both in this PR - maybe you want to always use one or another